saleor/payment/gateways/braintree/__init__.py

```diff
@@ -33,23 +30,23 @@
 }
 
 
-def get_customer_data(payment: Payment) -> Dict:
+def get_customer_data(payment: Dict) -> Dict:
     return {
-        "order_id": payment.order_id,
+        'order_id': payment['order'],
         'billing': {
-            'first_name': payment.billing_first_name,
-            'last_name': payment.billing_last_name,
-            'company': payment.billing_company_name,
-            'postal_code': payment.billing_postal_code,
-            'street_address': payment.billing_address_1,
-            'extended_address': payment.billing_address_2,
-            "locality": payment.billing_city,
-            'region': payment.billing_country_area,
-            'country_code_alpha2': payment.billing_country_code},
+            'first_name': payment['billing_first_name'],
```

Comment:
Why can't the payment dict have a sub-dict named `billing`?

Reply:
`payment` currently is just `model_to_dict(Payment)`, nothing fancy and those are normal fields of that model